pygmt/tests/test_basemap.py

```diff
@@ -48,9 +48,7 @@ def test_basemap_loglog():
 def test_basemap_power_axis():
     "Create a power axis basemap plot"
     fig = Figure()
-    fig.basemap(
-        R=[0, 100, 0, 5000], J="x1p0.5/-0.001", B='x1p+l"Crustal age"y500+lDepth'
-    )
+    fig.basemap(R=[0, 100, 0, 5000], J="x1p0.5/-0.001", B='x1p+l"Crustal age"')
```

Comment:
I think the original test wants to add labels for both X and Y axis. It should be changed to:
```suggestion
    fig.basemap(R=[0, 100, 0, 5000], J="x1p0.5/-0.001", B=['x1p+l"Crustal age"', 'y500+lDepth'])
```

Reply:
Ok, I'll change that in a minute. Will need to regenerate the plot too.